provider.go

```diff
@@ -301,6 +294,13 @@ func (p *Provider) MountSecretsStoreObjectContent(ctx context.Context, attrib ma
 	if tenantID == "" {
 		return fmt.Errorf("tenantId is not set")
 	}
+
+	azureCloudEnv, err := ParseAzureEnvironment(cloudName)
+	if err != nil {
+		log.Infof("cloudname is not valid")
+		return err
```

Comment:
`return fmt.Errorf("cloudName is not valid, error: %v", err)`

Reply:
Done